source/expressions.tex

```diff
@@ -4155,11 +4155,13 @@
 If the operand is a \grammarterm{qualified-id} naming a non-static or variant member \tcode{m}
 of some class \tcode{C} with type \tcode{T}, the result has type ``pointer to member
 of class \tcode{C} of type \tcode{T}'' and is a prvalue designating \tcode{C::m}.
-Otherwise, if the type of the expression is \tcode{T}, the result has type ``pointer to
-\tcode{T}'' and is a prvalue that is the address of the designated object\iref{intro.memory}
-or a pointer to the designated function. \begin{note} In particular, the address of an
-object of type ``\cv{}~\tcode{T}'' is ``pointer to \cv{}~\tcode{T}'', with the same
-cv-qualification. \end{note}
+Otherwise, if the type of the expression is \tcode{T},
+the result is a prvalue of type ``pointer to \tcode{T}''
```

Comment:
The use of the word "result" here is unfortunate, since "result" is also the name we give to the value produced by the expression. How about this:

"Otherwise, if the type of the operand is `T`, the resulting expression is a prvalue of type ''pointer to `T`'' whose result is a pointer to the designated object or function."

Reply:
Or maybe better (to avoid having two parallel-looking "if"s where one is restrictive and the other is not), how about:

"Otherwise, for an operand of type `T`, the resulting expression is a prvalue of type ''pointer to `T`'' whose result is a pointer to the designated object or function."

Reply:
Applied.